slurm/assets/logs/slurm_tests.yaml

```diff
@@ -0,0 +1,42 @@
+id: "slurm"
+tests:
+ -
+  sample: "[2024-10-22T23:16:26.830] debug:  REQUEST_PERSIST_INIT: CLUSTER:linux VERSION:9472 UID:0 IP:172.22.0.4 CONN:9"
```

Comment:
You can also add the service property in your samples [like done here](https://github.com/DataDog/integrations-core/blob/70f798dc68201e36078fa5b57c29308436c339a0/cisco_duo/assets/logs/cisco-duo_tests.yaml#L47)

Reply:
I am approving the PR as this is not extremely important, but if you have a bit of time, it can be a small improvement for the test file